src/ssl/support.cc

```diff
@@ -1172,7 +1195,11 @@ untrustedToStoreCtx_cb(X509_STORE_CTX *ctx,void *data)
     if (SquidUntrustedCerts.size() > 0)
         completeIssuers(ctx, sk);
 
+#if HAVE_LIBCRYPTO_X509_STORE_CTX_SET0_UNTRUSTED
```

Comment:
This looks like it should be getting the same renaming + definition treatment the other calls are given.

I have not investigated whether these two calls are actually drop-in identical. That will need to be checked before approval of this PR. If they are not, then this is fine but the gain or behaviour tradeoff between the two needs to be documented (here or in commit message).

Reply:
It's easy to see they're drop-in identicals by looking at the openssl 1.1 headers.  What the do is add a define for the old name when previous-versions API compatibility is needed.  You could do it either by using the new name or the old one (and libressl is keeping both defined).  I would prefer to use the new name in the main code, and add a define for it when using na old version.  With time, the old version should be dropped, and the main code will not need changes.